src/components/drawer/Swipeable.tsx

```diff
@@ -249,7 +257,7 @@ export default class Swipeable extends Component<Props, StateType> {
       rightThreshold = rightWidth / 2,
       fullLeftThreshold,
       fullRightThreshold,
-      onToggleSwipeLeft
+      onToggleSwipeLeft,
```

Comment:
Why is there a comma here? Doesn't it fail on lint tests?

Reply:
No, but I removed it.